### PR TITLE
[INLONG-7264][Dashboard] Support sink Apache Kudu

### LINKED ISSUE #7264: [Feature][Dashboard] Support sink Apache Kudu
### Description

_No response_

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by featzhang**
### Prepare a Pull Request

- [INLONG-7264][Dashboard] Support sink Apache Kudu

- Fixes #7264 

### Verifying this change

*(Please pick either of the following options)*

- [x] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

File: inlong-dashboard/src/locales/cn.json [hunk 1]
```diff
@@ -344,6 +344,17 @@
   "meta.Sinks.Redis.DataSeparator.Asterisk": "星号(*)",
   "meta.Sinks.Redis.DataSeparator.DoubleQuotes": "双引号(\")",
   "meta.Sinks.Redis.FormatDataSeparator": "数据字段分隔符",
+  "meta.Sinks.Kudu.masters": "masters",
```
Comment (healchow) [inline inlong-dashboard/src/locales/cn.json:347]: Did this `masters` need i18n?

File: inlong-dashboard/src/locales/cn.json [hunk 2]
```diff
@@ -344,6 +344,17 @@
...
+  "meta.Sinks.Kudu.TableName": "表名",
+  "meta.Sinks.Kudu.ExtListHelper": "kudu表的DDL属性需带前缀'ddl.'",
```
Comment (healchow) [inline inlong-dashboard/src/locales/cn.json:349]:
```suggestion
  "meta.Sinks.Kudu.ExtListHelper": "Kudu 表的 DDL 属性需添加'ddl.'前缀",
```

File: inlong-dashboard/src/locales/en.json [hunk 3]
```diff
@@ -344,6 +344,17 @@
   "meta.Sinks.Redis.DataSeparator.Asterisk": "Asterisk(*)",
   "meta.Sinks.Redis.DataSeparator.DoubleQuotes": "Double quotes(\")",
   "meta.Sinks.Redis.FormatDataSeparator": "Data separator",
+  "meta.Sinks.Kudu.masters": "masters",
```
Comment (healchow) [inline inlong-dashboard/src/locales/en.json:347]: ditto.

File: inlong-dashboard/src/locales/en.json [hunk 4]
```diff
@@ -344,6 +344,17 @@
...
+  "meta.Sinks.Kudu.TableName": "Table",
+  "meta.Sinks.Kudu.ExtListHelper": "The DDL attribute of the hudi table needs to be prefixed with 'ddl.'",
+  "meta.Sinks.Kudu.ExtList": "ExtList",
```
Comment (healchow) [inline inlong-dashboard/src/locales/en.json:350]:
```suggestion
  "meta.Sinks.Kudu.ExtListHelper": "The DDL attribute of the Kudu table needs to be prefixed with 'ddl.'",
  "meta.Sinks.Kudu.ExtList": "Extenstion params",
```

**Review by healchow — CHANGES_REQUESTED** [review]

**Review by leezng — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by healchow — APPROVED** [review]
LGTM

**Pull request merged**